scripts/build.js

```diff
@@ -4,8 +4,13 @@
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 const tasks = require('./tasks')
+const path = require('path')
 
-const buildDir = process.env.TARGET_GEN_DIR
+let buildDir = process.env.TARGET_GEN_DIR
+
+if (process.env.NODE_ENV === 'development') {
```

Comment:
we don't want to pass this in anymore in node, we should use gn instead and set it based on `is_component_build`